_summaries/gropp2020implicit_1.md

```diff
@@ -1,19 +1,55 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{gropp2020implicit}}
 # Please fill out info below
-author: # Your GitHub id
+author: mmcinnestaylor # Your GitHub id
 score: # How did you like this paper 0(dislike) to 10(love)
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Core idea?  
+* 2D and 3D Shapes can be represented as level sets of neural networks.    
+* Neural shape representations can be learned from raw data by using implicit geometric regularization.  
+* Implicit neural representations can be achieved without 3D supervision and/or a direct loss on the zero level set.     
+
+## Technical Implementation  
+* **Given**: a point cloud $$X = \{x_i\}_{i\in I}\subset\mathbb{R}^3$$ with or without normal data $$N =\{n_i\}_{i\in I}\subset\mathbb{R}^3$$  
+* **Goal**: compute the parameters $$\theta$$ of a Multilayer Perceptron(MLP) $$f(x;\theta)$$ where $$f:\mathbb{R}^3\times\mathbb{R}^m\rightarrow\mathbb{R}$$ so that it approximates a signed distance function to a surface $$M$$ defined by $$X\text{ and }N$$.  
+![2D](gropp2020implicit_1_a.png)  
+![3D](gropp2020implicit_1_b.png)   
+* **Using loss**: $$l(\theta) = l_x(\theta) + \lambda\mathbb{E}_x(\Vert\nabla_xf(x;\theta)\Vert-1)^2$$  
+	* Note: $$\lambda>0$$  is a parameter, $$\Vert\cdot\Vert=\Vert\cdot\Vert_2$$ is the euclidean 2-norm    
+	* The first term: $$l_x(\theta)=\frac{1}{I}\sum_{i\in I}(\lvert f(x_i;\theta)\rvert+\tau\Vert\nabla_xf(x_i;\theta)-n_i\Vert)$$  
+		* encourages $$f$$ to vanish on $$X$$ and if normal data exists, that $$\nabla_xf$$ is close to the normals $$N$$    
+	* The second term is the Eikonal term, and encourages $$\nabla_xf$$ to be of unit 2-norm  
+		* The expectration is taken w.r.t. some probability distribution in $$\mathbb{R}^3$$  
```

Comment:
Consider mentioning the plane reproduction property of the loss here (reaches minima rather than maxima or saddle according to proof in section 4). Consider mentioning the implicit geometrical regularization as a consequence of plane reproduction (chooses a smooth solution out of infinite solutions).